spotifython/user.py

```
@@ -0,0 +1,332 @@
+from album import Album
+from artist import Artist
+from player import Player
+from playlist import Playlist
+from spotifython import Spotifython
+from track import Track
+
+from response import Response
+from typing import Union, List, Dict
+
+# TODO: exceptions for each param
```

Should throw an exception any time the user tries to go above the Spotify internal API limit.